### PR TITLE
[skip-ci][Core] Update TMethodCall documentation

### PR DESCRIPTION
**Pull request by dpiparo**
fixes [ROOT-6968](https://its.cern.ch/jira/browse/ROOT-6968)

### CODE REVIEW

File: core/meta/src/TMethodCall.cxx [hunk 1]
```diff
@@ -580,8 +580,7 @@ TMethodCall::EReturnType TMethodCall::ReturnType()
 
 ////////////////////////////////////////////////////////////////////////////////
 /// ParamArr is an array containing the function argument values.
```
Comment (ferdymercury) [inline core/meta/src/TMethodCall.cxx:582]:
```suggestion
/// \brief Set pointers to parameters
/// \param paramArr an array containing the function argument values.
```

File: core/meta/src/TMethodCall.cxx [hunk 2]
```diff
@@ -580,8 +580,7 @@ TMethodCall::EReturnType TMethodCall::ReturnType()
...
-/// If nparam = -1 then paramArr must contain values for all function
-/// arguments, otherwise Nargs-NargsOpt <= nparam <= Nargs, where
+/// Nargs-NargsOpt <= nparam <= Nargs, where
```
Comment (ferdymercury) [inline core/meta/src/TMethodCall.cxx:583]:
```suggestion
/// \param nparam number of parameters should fulfill Nargs-NargsOpt <= nparam <= Nargs, where
```

**Review by ferdymercury — CHANGES_REQUESTED** [review]
Thanks, LGTM.

Please consider also fixing the typos on line 549 ("actual" is missing last char, and the closing parenthesis is missing).

**Comment by dpiparo** [comment]
All implemented. Thanks a lot.

**Pull request merged**